### PR TITLE
[polskieradio] Add thumbnails.

### PR DESCRIPTION
**Pull request by JakubAdamWieczorek**
### What is the purpose of your *pull request*?
- [x] Bug fix
- [ ] New extractor
- [ ] New feature

---

### Description of your *pull request* and other information

This is a simple change to extract thumbnails from Polskie Radio auditions.

### CODE REVIEW

**Comment by yan12125** [comment]
Usually the ```thumbnail``` field in ```_TESTS``` uses regular expressions instead of full URLs. See https://github.com/rg3/youtube-dl#adding-support-for-a-new-site for an example.

**Comment by JakubAdamWieczorek** [comment]
@yan12125 Well, it's important we check that it finds the right image. Especially on Polskie Radio the page often contains, aside from the main image, multiple thumbnails for related materials.

**Comment by yan12125** [comment]
To avoid any possible false-positive, the extraction method itself should be precise. In this case, it's unlikely that ```_og_search_thumbnail``` finds a wrong image as in normal cases there's only one ```og:image``` tag in a web page.

**Comment by JakubAdamWieczorek** [comment]
Agree, yet the test should not be concerned with the implementation details.

**Comment by yan12125** [comment]
You want to make sure that the extractor always gives the correct thumbnail, but using a full URL does not increase the detection rate of website changes. Thumbnail URLs are constantly changing. If there's a failed test, we can't know, without further investigation, whether it's due to wrong files or just moved/renamed files. Though I don't have a concrete statistical result, moved files are more common than wrong files, and in this case, a moved/renamed file is almost the only possible reason. As a result, if there's a failed test in polskieradio in the future, we will just update the thumbnail URL. It wastes time of developers, so I recommend you to use a regular expression.

If you really want a strict check, the correct way is to check the MD5 checksum of thumbnails, which is much less unlikely to change.

**Comment by JakubAdamWieczorek** [comment]
That's a fair point on the MD5, thank you. I made the suggested change. :)

File: youtube_dl/extractor/polskieradio.py [hunk 1]
```diff
@@ -87,6 +90,7 @@ def _real_extract(self, url):
                 'duration': int_or_none(media.get('length')),
                 'vcodec': 'none' if media.get('provider') == 'audio' else None,
                 'timestamp': timestamp,
+                'thumbnail': thumbnail_url
```
Comment (TRox1972) [inline youtube_dl/extractor/polskieradio.py]: Just pass in `self._og_search_thumbnail()` directly here, since it is not used anywhere else

**Comment by yan12125** [comment]
Well ```md5:``` checks for the field value but not the corresponding file contents. You'll have to extend ```test/test_download.py```

**Comment by JakubAdamWieczorek** [comment]
@yan12125 That seems like a good idea, sadly I'm rather time constrained now. I've gone with the regular expression approach for now.

Is this good to go now?

**Comment by yan12125** [comment]
Thanks. Merged with minor fixes.

**Comment by JakubAdamWieczorek** [comment]
Ah, of course. Ought to have escaped the dots. Thanks!

**Pull request merged**